content/intro-to-reading-data.md

```diff
@@ -44,9 +51,9 @@ In this lesson we won’t consider much about accounts beyond their ability to s
 
 ### Public Keys
 
-Public keys are often referred to as addresses. The addresses point to accounts on the Solana network. If you want to run a specific program or transfer SOL, you’ll need to provide the necessary public key (or keys) to do so.
+Public keys are often referred to as addresses. The addresses point to accounts on the Solana network. If you want to run a specific program a.k.a smart contract or just want to transfer some SOL, you’ll need to provide the necessary public key (or keys) to do so.
```

Comment:
'Smart contract' is not used on Solana. https://solana.com/docs/terminology#onchain-program